ivy-data/src/main/java/com/ivy/data/db/entity/LoanEntity.kt

```diff
@@ -37,7 +37,7 @@ data class LoanEntity(
     val isDeleted: Boolean = false,
     @SerialName("dateTime")
     @Serializable(with = KSerializerLocalDateTime::class)
-    val dateTime: LocalDateTime,
+    val dateTime: LocalDateTime? = null,
```

Comment:
If we're using "loans_new" this table must also be renamed. The migration overall seems very.risky and I doubt it works in all cases

Reply:
"loans_new" is just for creating a new table. The new table is replacing the old one with the same name "loans", at the end of the migration.

Reply:
Got it - I'll test it. Revert `124.json` to be like the one in main

Reply:
> Got it - I'll test it. Revert `124.json` to be like the one in main

I am a bit confused why should we revert `124.json`?